AboutPage.html

```diff
@@ -0,0 +1,123 @@
+        </ul>
+      </header>
+
+  <main>
+    <section class="mobile-menu" id="mobileNave">
+      <ul class="nav-list2"> 
+        <img onclick="closeNave();" class="img-close" src="./images/Icon_X_.png" alt="">
+        <li class="mobile-link" onclick="closeNave();"><a href="AboutPage.html" >About</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#aboutme-container" >Program</a></li>
+        <li class="mobile-link" onclick="closeNave();"><a href="#contact">News</a></li>
+    </section>
+
+    <section class="home-about" id="home-about" >
+      <div class="containerHome-Page2">
+        <div>
+          <h1 class="about-title" id="about-title" >Urban Art Festival <br>Creativity in the Streets 2023</h1>
+        </div>
+        <div class="home-Description">
+          <p>
+            Welcome to the Urban Art Festival "Creativity in the Streets," where imagination takes to the streets and creativity knows no bounds. At the festival, we are dedicated to bringing urban art to life and creating a vibrant platform for artists, enthusiasts, and culture lover, Founded with a vision to transform the city into a canvas of artistic expression, the Urban Art Festival "Creativity in the Streets" was born. From its inception in [año de fundación], our festival has evolved into an annual celebration that redefines the way we experience art in urban spaces. Our journey has been marked by [menciona algunos hitos clave, por ejemplo: "awe-inspiring murals, dynamic live performances, and a growing community of artists"
+          </p>
+        </div>
+        <div>
+          <p class="home-date2">Please contact us per Email for nay further questions about Urban Art Festival "Creativity in the Streets</p>
+          <p class="home-location-info2">UrbanArtFestival2023@GMAIL.COM </p>
+        </div>
+      </div>
+    </section>
+    <section class="logoAboutSection">
+      <h2>Urban Art Festival "Creativity in the Streets</h2>
+      <hr>
+      <h4>At the heart of the Urban Art Festival "Creativity in the Streets" lies a powerful emblem that encapsulates the essence of our event—the logo. Crafted with intention and innovation, our logo is more than just a visual mark; it's a window into the world of urban art, where imagination and expression collide.</h4>
+      <img src="/images/LogoUrban.png" alt="" srcset="">
+    </section>
+
+    <section class="historySection">
+      <h2>Urban Art Festival "Creativity in the Streets</h2>
+      <hr>
+      <h4>At the heart of the Urban Art Festival "Creativity in the Streets" lies a powerful emblem that encapsulates the essence of our event—the logo. Crafted with intention and innovation, our logo is more than just a visual mark; it's a window into the world of urban art, where imagination and expression collide.</h4>
+      <div class="imageHistory">
+        <img src="/images/festival01.jpg" alt="" srcset="">
+        <img src="/images/festival02.jpg" alt="" srcset="">
+      </div>
```

Comment:
- [ ] Could you please take a moment to review your images? It seems like they have varying sizes.
- [ ] It would be great if you could add the appropriate text within each of your images, following the example in the template. This will help everything look consistent and neat.

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/cbf2229e-50ea-49be-8a88-5a6f2e0b19de)
  
   ### Behance design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/b17b2a49-b422-41ff-a17e-9ac3659743cc)